cmd/build/helmify/static/values.yaml

```diff
@@ -70,3 +70,4 @@ pdb:
   controllerManager:
     minAvailable: 1
 service: {}
+builtinHttpSend:
```

Comment:
We probably want to name this something like `disabledBuiltins` and allow it to take a list of disabled builtins. Helm chart users may want to disable more than HTTP send

Reply:
+1 on naming and we need a default

Reply:
Changed the variable into a list, that solves the unset `--disable-opa-builtin` for us.